src/components/ConsultForm/ConsultForm.jsx

```diff
@@ -0,0 +1,102 @@
+import { Form, Section } from 'components';
+import cn from 'classnames';
+import {
+	Input,
+	TextArea,
+	Typography,
+	Checkbox,
+	Label,
+	Button,
+	LinkComponent,
+	PhoneInput,
+} from 'shared/ui';
+import styles from './ConsultForm.module.scss';
+
+function ConsultForm() {
+	return (
+		<Section>
+			<div className={styles.container}>
+				<Typography variant={'h2'}>Заявка на консультацию</Typography>
+				<Typography variant={'body2'}>
+					Если у вас возникли вопросы по страхованию, вы можете оставить заявку,
+					и наши специалисты свяжутся с вами в течение 1 часа.
+				</Typography>
+			</div>
+
+			<Form type="consult-form">
+				<div className={cn(styles.box, styles.inputs)}>
+					<div className={cn(styles.box, styles.input)}>
+						<Label mode="default" inputId="nameConsult" text="Имя*" />
+						<Input
+							name="nameConsult"
+							id="nameConsult"
+							type="text"
+							mode="default"
+							isValid={true}
+							required={true}
+						/>
+					</div>
```

Comment:
1. Компоненты очень схожи - лучше их декомпозировать
2. У вас много где используется mode="default" такое нужно выносить в const
3. Возможна ли ситуация когда у вас mode имеет разные значения?
```suggestion
					<div className={cn(styles.box, styles.input)}>
						<Label mode="default" inputId="nameConsult" text="Имя*" />
						<Input
							name="nameConsult"
							id="nameConsult"
							type="text"
							mode="default"
							isValid
							required
						/>
					</div>
```